docs: updating minikube and crc installation guides (#2302)
codenvy/che-docs: modules/administration-guide/partials/proc_using-chectl-to-install-che-on-codeready-containers.adoc

.Procedure

> .. Remove any previous cluster:

Reviewer: ```suggestion
```

+
----

> $ crc delete

Reviewer: crc setup first.
But as it's part of the crc documentation, we could rely on it and skip the `crc setup ; crc delete` steps in our docs.

Author: why crc setup first? 
if there is an instance already, first you should delete it

Reviewer: See: 
https://console.redhat.com/openshift/create/local
https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/2.0/html-single/getting_started_guide/index#setting-up-codeready-containers_gsg

successful `crc setup` is a prerequisite to run other `crc` commands `crc start` or `crc delete`

Reviewer: These steps are covered by the prerequisites: An installation of CodeReady Containers. We should remove them.

Reviewer: If there is an instance already, it means you already ran `crc setup`.

Author: well, I do not think we should remove them. Currently, the procedure is verified and very well defined. If you have any concerns please provide the GH suggestion.

Reviewer: ```suggestion
```

Reviewer 2: `crc delete` makes perfect sense to run before `crc setup`, @themr0c. The `delete` subcommand **does not** depend on the existence of files created during the `setup` command. As long as the `crc` executable exists in `$PATH`, you can safely run `crc delete`. If the instance doesn't exist already, no worries -- if it does, it deletes it. I believe this is safe to include in a general procedure.

----

> . Initiate the CRC platform:

Reviewer 2: Hmm... this is an issue for me. (I'm the primary docs writer for CodeReady Containers/OpenShift Local, FWIW.)

From a documentation perspective, we should not use the CRC acronym. Always use the full name, CodeReady Containers. Unfortunately, we also happen to be going through a rename process (to "CRC" -- not "CodeReady Containers" -- upstream, "OpenShift Local" downstream) at the moment, so the name CodeReady Containers will not last much longer.

For this reason, perhaps we should consider replacing *all* occurrences of "CRC" with "CodeReady Containers". This would allow the team (in the future) to replace the existing content with the new name more easily (using `sed`, perhaps?).

Having the same thing referred to with multiple names (or abbreviations and acronyms) makes this task more difficult.

Reviewer 2: TL;DR: My recommendation at the moment is simply to use the full name, CodeReady Containers.

Reviewer: ```suggestion
. Initiate the CodeReady Containers platform:
```

> . Initiate the CRC platform:
> 
> .. Configure your host machine for CodeReady Containers:

Reviewer: ```suggestion
. Set up prerequisites for using CodeReady Containers, and delete any previous instance.
```

+
----

> $ crc setup

Reviewer: ```suggestion
$ crc setup
$ crc delete
```

Reviewer 2: I would reverse these directions, personally. We want to delete an existing instance if it already exists.

Note that the `crc cleanup` command also exists to ensure that changes made during `crc setup` are, more or less, undone. If you want to **completely** remove a previous instance (or installation in general), the procedure looks like this:

```
$ crc delete
$ crc cleanup
$ crc setup
```

Author: fully agree :+1:

----

> .. Initiate the `oc` command line interface within CRC: 

Reviewer: ```suggestion
. Initiate the `oc` command line interface within CRC: 
```

Reviewer 2: This is really more like "allowing access to" than "initiating".

Also worth noting that this isn't just "within CRC". This allows access to the embedded `oc` executable contained within CodeReady Containers for the _entire shell_ at that moment.

Reviewer 2: ```adoc
. Enable access to the `oc` command line interface embedded in CodeReady Containers:
```

Reviewer: ```suggestion
.. Enable access to the `oc` command line interface embedded in CodeReady Containers:
```

.Procedure

.. Remove any previous cluster:
+
----
$ crc delete
----

. Initiate the CodeReady Containers platform:

.. Configure your host machine for CodeReady Containers:
+
----
$ crc setup
----

> .. Enable access to the `oc` command line interface embedded in CodeReady Containers:

Reviewer: Should happen *after* crc start, or you receive the error:

```
eval $(crc oc-env)
Machine does not exist. Use 'crc start' to create it
```

Reviewer: Suggestions in https://github.com/ibuziuk/che-docs/pull/3

*    Use CodeReady containers rather than CRC
*    Fix steps numbering.
*  Tested the procedure: removing obsolete commands, reordering commands.

Author: @themr0c applied, thanks
